### ISSUE TITLE
fcntl package missing in Windows Python

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#138]
```
[Use this form for both apiclient and oauth2client issues]

What steps will reproduce the problem?
1. Install Python for Windows
2. Import oauth2client.multistore_file

What is the expected output? What do you see instead?

oauth2client should "work." but instead throws an import error.

What version of the product are you using? On what operating system?

Windows 7.  Python 2.6.4.

Please provide any additional information below.

```

Original issue reported on code.google.com by `ca...@google.com` on 31 May 2012 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
Code out for review http://codereview.appspot.com/6265043/
```

Original comment by `jcgregorio@google.com` on 6 Jun 2012 at 6:26
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I am also facing same issue. Kindly do the needful asap.
```

Original comment by `singham...@gmail.com` on 7 Jun 2012 at 10:44

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision ac147884bf0c.
```

Original comment by `jcgregorio@google.com` on 13 Jun 2012 at 4:19
- Changed state: **Fixed**
